com/truckmuncher/api/trucks.proto

```diff
@@ -30,11 +30,20 @@ service TruckService {
     /**
      * Modify the serving mode for a truck.
      *
-     * This call requires Vendor authorization
+     * This call is an alias for reportServingMode
+     * This call is deprecated and should no longer be used by clients. It will be removed in the near future.
+     * This call requires the user to be logged in
      */
     rpc modifyServingMode (ServingModeRequest) returns (ServingModeResponse);
```

Comment:
I made the comment "deprecated" because the name of the method wasn't good for what it does anymore. "reportServingMode" would be a better description of what's happening. That's the route everything will use for reporting.